source/includes/fact-legacy-connections.rst

```diff
@@ -4,5 +4,6 @@ internal |compass-short| connection-options format that is stored on disk and no
 longer supported after version 1.39.0. 
     
 If you have legacy connections saved in your favorites, export the 
-connections on version 1.39.0 to convert them into the new format before 
-updating to version 1.39.2.
+connections on `version 1.39.0 
+<https://github.com/mongodb-js/compass/releases/tag/v1.39.0>`_ to convert them 
+into the new format before updating to version 1.39.2.
```

Comment:
[question]

Should we say "before updating to version 1.39.2 *or later*" ?

Reason being is that now that this note is no longer just on the release notes, we'll have users reading this behavior on versions well past version 1.39.2, and it's not totally clear that this would apply to them.

Reply:
Ah gotcha, that makes sense, especially with 1.39.4 having just released, too. Updating!